naga/src/lib.rs

```diff
@@ -2271,4 +2272,10 @@ pub struct Module {
     pub functions: Arena<Function>,
     /// Entry points.
     pub entry_points: Vec<EntryPoint>,
+    /// Arena for all diagnostic filter rules parsed in this module, including those in functions.
+    pub diagnostic_filters: Arena<DiagnosticFilterNode>,
+    /// The head of a linked list of `diagnostic(…)` directives parsed in this module.
+    ///
+    /// TODO: doc more
+    pub diagnostic_filter_head: Option<Handle<DiagnosticFilterNode>>,
```

Comment:
Similarly, I think `diagnostic_filter_leaf` is a better name here.

Reply:
Fixed in the force-pushed d581c157.